jsonschema/jsonschema.go

```diff
@@ -0,0 +1,13 @@
+package jsonschema
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"github.com/invopop/jsonschema"
+)
+
+func Generate(a any) ([]byte, error) {
```

Comment:
Since this is a public function, I think a comment above explaining what the expected input/output is (or where to read up about it) would be good

Reply:
done in 25fe25c